pkg/clients/k8s/client.go

```diff
@@ -5,12 +5,16 @@ import (
 
 	"github.com/rancher/wrangler/pkg/clients"
 	v1 "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	authnv1 "k8s.io/client-go/kubernetes/typed/authentication/v1"
+	authzv1 "k8s.io/client-go/kubernetes/typed/authorization/v1"
 	"k8s.io/client-go/rest"
 )
 
 type Clients struct {
 	ConfigMaps v1.ConfigMapClient
 	Secrets    v1.SecretController
+	SAR        authzv1.SubjectAccessReviewInterface
+	TR         authnv1.TokenReviewInterface
```

Comment:
Would be better to spell these out in full rather than abbreviate

```suggestion
	SubjectAccessReviews        authzv1.SubjectAccessReviewInterface
	TokenReviews         authnv1.TokenReviewInterface
```

Reply:
Resolving - support config functionality and all related auth was removed from this PR.